scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py

```diff
@@ -261,7 +262,7 @@ def make_extract_table(fq_ranges_dataset,
 
     ## TODO -- provide a cmdline arg to override this (so we can simulate smaller datasets)
 
-    global PET_VET_TABLE_COUNT
+    global PET_VET_TABLE_COUNT ## TODO why are we using PET here?
```

Comment:
I think it's just a name, and now it would more accurately be REF_VET_TABLE_COUNT